Minor rewording of ASM to ARM migration unsupported configuration

caseywatson opened the pull request:
Unsupported configuration "Virtual networks that contain virtual machines and web/worker roles" language was confusing to customer. They incorrectly assumed that it meant that virtual networks must not contain either. Minor update to state that "Virtual networks that contain [both] virtual machines and web/worker roles" is an unsupported configuration.

azurecla commented:
Hi __@siriusk-MSFT__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

caseywatson committed:
Minor rewording of ASM to ARM migration unsupported configuration

Unsupported configuration "Virtual networks that contain virtual machines and web/worker roles" language was confusing to customer. Minor update to state that "Virtual networks that contain [both] virtual machines and web/worker roles"
